doc/ansible/dev/testing_guide.md

```diff
@@ -6,9 +6,8 @@
 To begin, you sould have:
 - The latest version of the [operator-sdk](https://github.com/operator-framework/operator-sdk) installed.
 - Docker installed and running
-- [Molecule](https://github.com/ansible/molecule) >= v2.20
-- [Ansible](https://github.com/ansible/ansible) >= v2.7
-- [jmespath](https://pypi.org/project/jmespath/)
+- [Molecule](https://github.com/ansible/molecule) >= v2.22
+- [Ansible](https://github.com/ansible/ansible) >= v2.9
```

Comment:
It was outdated. Just to confirm.

Reply:
Yeah, now that we are using a collection for the `k8s_status` module 2.9 is the absolute minimum we can support.